tests/integration/filterable/filterable_core.js

```diff
@@ -6,6 +6,7 @@
 module( "Filter Widget Core Functions" );
 
 asyncTest( "Filter hides/shows results when the user enters information", function() {
+	$( "body" ).enhance();
```

Comment:
This assumes that the first test will run first. Why is this necessary at all? The test page includes init, so the page should be enhanced by the time the first test starts to execute. This is a good test case for making sure the enhancer works correctly in a requirejs scenario. Please have a look at the index.html in this folder! The modules it loads should be sufficient for getting a fully enhanced page by the time the first of these tests runs. In fact, we shouldn't even need to load the page module first. Furthermore, we shouldn't need to load the page module explicitly at all, because init should be pulling it in and starting up navigation, which should cause the page to be enhanced.